linter_exclusions.yml

```diff
@@ -2027,24 +2027,24 @@ network vwan update:
     security_provider_name:
       rule_exclusions:
       - option_length_too_long
-nf controller create:
+networkfabric controller create:
   parameters:
     workload_er_connections:
       rule_exclusions:
       - option_length_too_long
-nf rack create:
+networkfabric rack create:
   rule_exclusions:
   - missing_command_test_coverage
```

Comment:
Why not adding test for the rack commands?

Reply:
We are not creating Rack separately. Network Fabric resource will create the Rack.